### PR TITLE
Added Support for managed-schema in 'solr.EnumerateValidationResults()'

### PR DESCRIPTION
**Pull request by BkrBkr**
Hello,

the function 'solr.EnumerateValidationResults()' was designed to work with the 'schema.xml' file only.  This patch should work with both: the schema.xml and the managed-schema file. BUT I didn't tested it with the schema.xml. For some unknown reason I can't debug the 'MappingValidator' so I didn't tested it either. But the results are looking good.

Thank You

### CODE REVIEW

File: SolrNet/Impl/SolrServer.cs [hunk 1]
```diff
@@ -314,9 +314,9 @@ public SolrSchema GetSchema(string schemaFileName)
             return basicServer.GetSchema(schemaFileName);
         }
 
-        public IEnumerable<ValidationResult> EnumerateValidationResults()
+        public IEnumerable<ValidationResult> EnumerateValidationResults(String schemaFileName)
```
Comment (gjunge) [inline SolrNet/Impl/SolrServer.cs]: This method is an implementation of `ISolrOperations`, so 
- add `IEnumerable<ValidationResult> EnumerateValidationResults(string schemaFileName);` to ISolrOperations

It also shouldn't be a breaking change, so keep the non-parameterized method in place.

File: SolrNet/Schema/SolrSchemaParser.cs [hunk 2]
```diff
@@ -37,12 +37,16 @@ public class SolrSchemaParser : ISolrSchemaParser {
 
             var schemaElem = solrSchemaXml.Element("schema");
 
-            foreach (var fieldNode in schemaElem.XPathSelectElements("types/fieldType|types/fieldtype")) {
+            foreach (var fieldNode in schemaElem.XPathSelectElements("types/fieldType|types/fieldtype|fieldtype|fieldType")) {
```
Comment (gjunge) [inline SolrNet/Schema/SolrSchemaParser.cs]: Could you back up this code with one or two unit tests?
See `SolrSchemaParserTests.cs`

Comment (BkrBkr) [reply SolrNet/Schema/SolrSchemaParser.cs]: I'll do, but give me time until next week

Comment (BkrBkr) [reply SolrNet/Schema/SolrSchemaParser.cs]: I did the requested changes. I hope I did the merge well. Was my first time. Do I have to mark your requests as solved somewhere?

**Comment by BkrBkr** [comment]
I did the requested changes. I hope I did the merge well. Was my first time. Do I have to mark your requests as solved somewhere?

**Comment by gjunge** [comment]
@BkrBkr  Thanks, for the changes. I'll merge it now with our Develop branch, so that it will be part of our next 1.0.0 release.

**Pull request merged**